module/x/gravity/keeper/signer_set_test.go

```diff
@@ -0,0 +1,102 @@
+package keeper
+
+import (
+	"testing"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/peggyjv/gravity-bridge/module/v3/x/gravity/types"
+)
+
+func TestSignerSetTxExecuted(t *testing.T) {
+	input := CreateTestEnv(t)
+	ctx := input.Context.WithBlockHeight(100)
+	// storeKey := input.GravityStoreKey
```

Comment:
Should we be using this commented out code, or should we delete it?

Reply:
Ah, needs to be removed. Since timeouts don't exist for signer sets, the eth height isn't needed in state.